kuksa_viss_client/kuksa_viss_client/__main__.py

```diff
@@ -409,7 +405,19 @@ def do_printTokenDir(self, args):
 
 # Main Function
 def main():
-    clientApp = TestClient()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--ip', help="VISS Server IP Address", default=DEFAULT_SERVER_ADDR)
+    parser.add_argument('--port', type=int, help="VISS Server Port", default=DEFAULT_SERVER_PORT)
+    parser.add_argument(
+        '--protocol',
+        help="VISS Server Communication Protocol",
+        choices=SUPPORTED_SERVER_PROTOCOLS,
```

Comment:
We should better somewhere give a short description of the protocols, possibly in the README, like one line for ws and one for grpc. For ws we might refer to https://github.com/eclipse/kuksa.val/tree/master/doc/protocol but I do not know if it is fully up to date.

Reply:
I will keep this for the next PR where I integrate the new gRPC API. This way I will add a section to link the newer gRPC API.